agent/calculation/shared_utils.py

```diff
@@ -14,6 +14,11 @@
 
 logger = agentlogging.get_logger('dev')
 
+rdf_type_to_unit = {
+    constants.TRAJECTORY_COUNT: '[-]',
+    constants.TRAJECTORY_AREA: 'm²'
```

Comment:
This seems unexpected. One of the units includes the square brackets, but the other one doesn't? In my view, the unit string itself should not include the square brackets...

Reply:
These strings are directly displayed on the visualisation, if I don't add the square brackets here I'll have to add them in another place (e.g. the feature info agent)

Reply:
In that case I would find a way that wraps all the units (apart from empty strings) consistently with square brackets before passing them to the visualisation. That, btw, is another reason to define them in one place in the code, so you don't have to do this multiple times (which would then be at risk of being inconsistent and getting out of sync)...